### ISSUE TITLE
/dev/null: restricted: cannot redirect output

### ISSUE THREAD

**Issue by encryptedxdata** [hackerschoice/segfault#49]
Creating Server ShadowJar.............................../bin/segfaultsh: line 14                                                                                                                                                             5: /dev/null: restricted: cannot redirect output

howw to fix?

**Comment by SkyperTHC** [comment]
hmm. do you still get this?

Please run ssh -v and let us know which IP you are connecting to.

**Comment by encryptedxdata** [comment]
after updating to newest updates and enabling debug i got this
```
######################################################################
#### DISCLAIMER: TO BE USED FOR CREATIVE AND GOOD PURPOSES ONLY.. ####
#### TO TINKER AND TO EXPLORE.     >>>USE AT YOUR OWN RISK<<<     ####
######################################################################
Creating Server EnableExpect........................DEBUG: LID= SF_HOSTNAME=EnableExpect
...
docker: Error response from daemon: failed to create shim: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/root/segfault/config/etc/sf/WARNING---SHARED-BETWEEN-ALL-SERVERS---README.txt" to rootfs at "/everyone/WARNING---SHARED-BETWEEN-ALL-SERVERS---README.txt": open /var/lib/docker/overlay2/somehash.../merged/everyone/WARNING---SHARED-BETWEEN-ALL-SERVERS---README.txt: read-only file system: unknown.
ERROR: Failed to set up guest instance...
```

**Comment by SkyperTHC** [comment]
fixed in 0.3.7a
